delfin/drivers/utils/ssh_client.py

```diff
@@ -1,5 +1,6 @@
 # Copyright 2020 The SODA Authors.
 # Copyright (c) 2016 Huawei Technologies Co., Ltd.
+# Copyright 2011 OpenStack LLC
```

Comment:
Do we need these 3 organizations' copyright?

Reply:
Changed